.changeset/blue-socks-doubt.md

```diff
@@ -0,0 +1,17 @@
+---
+'astro': minor
+---
+
+Adds experimental support for built-in SVG components.
+
+After enabling the `experimental.svg` flag, `.svg` files can be imported and used as components. They will be inlined into the HTML output.
+
+```astro
+---
+import Logo from './path/to/svg/file.svg';
+---
+
+<Logo size={24} />
+```
+
+To learn more, check out [the documentation](https://docs.astro.build/reference/configuration-reference/#experimentalsvg).
```

Comment:
``````suggestion

This feature allows you to import SVG files directly into your Astro project as components. By default, Astro will inline the SVG content into your HTML output.

To enable this feature, set `experimental.svg` to `true` in your Astro config:

```js
{
  experimental: {
    svg: true,
  },
}
```

To use this feature, import an SVG file in your Astro project, passing any common SVG attributes to the imported component. Astro also provides a `size` attribute to set equal `height` and `width` properties:

```astro
---
import Logo from './path/to/svg/file.svg';
---

<Logo size={24} />
```

For a complete overview, and to give feedback on this experimental API, see the [Feature RFC](https://github.com/withastro/roadmap/pull/1035).
`````` 

For a new feature, we do like to hype it up a bit! The changeset is a great place to show both the config flag needed and usage. I think something like this works!